src/app/LoginController.java

```diff
@@ -75,8 +85,14 @@ public void run() {
     * @param ActionEvent when button clicked
     */
    public void login(ActionEvent event) throws IOException {
-      boolean isLoggedIn = true;
-
+       try {
+            this.loginDao = AccountDaoFactory.getDao();
+        } catch (AccountDaoException ex) {
+            Logger.getLogger(LoginController.class.getName()).log(Level.SEVERE, null, ex);
+            return;
```

Comment:
Return isn't necessary, especially as this is void so nothing is returned. I don't know what Cantor wants in style but I'd remove this.